src/System.Runtime/tests/System/StringTests.cs

```diff
@@ -23,6 +23,7 @@ public static class StringTests
         [InlineData(new char[] { 'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', '\0', 'i', 'j' }, "abcdefgh")]
         [InlineData(new char[] { 'a', '\0' }, "a")]
         [InlineData(new char[] { '\0' }, "")]
+        [InlineData(new char[] { '?', '@', ' ', '\0' }, "?@ ")] // ? and @ don't have overlapping bits
```

Comment:
Oh, when you wrote "current implementation" above, you meant  what's  in your PR rather than what's currently checked in?